assets/javascript/game.js

```diff
@@ -0,0 +1,194 @@
+    var wins = 0
+
+    //Create a function for the score
+    function scoreCal() {
+        //I compare the lenght of the word selected with the numer of points the aquire whrn they win
+        if (computerSelectionArrLines === computerSelectionArr) {
+            alert("YOU WON!")
+            wins++
+
+        }
+        else if (guessLeft = 0) {
+            alert("YOU LOOSE!")
+            wins++
+        };
+    }
+
+
+
+    //Create an empty array that we will fill with ---
+    var answerArr = [];
+
+
+
+    //Create a function that will add points if the word is completed ( maybe )
+
+
+
+
+    //when press stat the game begins and the function of random selection starts running
+    $("#start").click(function (event) {
+
+        //Change the start button to re-start
+        $(this).text("Play Again").addClass("grey")
+
+        //transform the bottom from start to refresh
+        $(this).click(function () {
+            location.reload();
+        })
+
+        //Create a variable that selects a random number inside our array
+        var arrNum = Math.floor(Math.random() * desingers.length)
+        //Relate that number with a person inside our array
+        var computerSelection = desingers[arrNum];
+
+
+
+        //Array with ---
+        var computerSelectionArrLines = [];
+        //Array with the index of the number
+        var computerSelectionArrIndex = [];
+
+        //create a for loop that will go through all the letters and change them to - and the space to ""
+        for (var i = 0; i < computerSelection.length; i++) {
+
+            if (computerSelection[i] === " ") {
+                // Add space into HTML (search the solution on google with meetup team)
+                answerArr[i] = "&nbsp;";
+            }
+            else {
+                answerArr[i] = "-";
+                //I am pushing the  index of the letter of the array and the position of it
+                computerSelectionArrLines.push(answerArr[i]);
+                computerSelectionArrIndex.push(i);
+            }
+
+        }
+
+
+
+        console.log("computerSelectionArrIndex " + computerSelectionArrIndex)
+        console.log("computerSelection" + computerSelection)
+
+        //Append the answer in the code so iI can change its positions
+        $("#wordGuess").append(computerSelectionArrLines);
+
+        //Create an array that will print the number of index of of those elements
+
+
+        //Create a function that will change those
+
+
+        computerSelectionArr = []
+        computerSelectionArr.push(computerSelection[arrNum]);
+
+
+
+
+
+
+
+
+
+
+        //Checking the letters inside the string selected
+        function checkingLetter(selection) {
+            for (var i = 0; i < computerSelection.length; i++) {
+                // Found charAt in w3 school "The charAt() method returns the character at the specified index in a string.""
+                //Ask? why do we need to add string if it was a string before.
```

Comment:
Great question, charAt() specifically looks for a character index in a string and will return empty if it is something else